src/SignalR.Orleans/HostingExtensions.cs

```diff
@@ -14,23 +19,7 @@ public static ISiloBuilder UseSignalR(this ISiloBuilder builder, Action<SignalrO
 
             cfg.ConfigureBuilder?.Invoke(builder, new HostBuilderConfig());
 
-            try
-            {
-                builder.AddMemoryGrainStorage(Constants.PUBSUB_PROVIDER);
-            }
-            catch
-            {
-                /** PubSubStore was already added. Do nothing. **/
-            }
-
-            try
-            {
-                builder.AddMemoryGrainStorage(Constants.STORAGE_PROVIDER);
-            }
-            catch
-            {
-                /** Grain storage provider was already added. Do nothing. **/
-            }
+            builder.ConfigureServices(services => services.AddSingleton<IConfigurationValidator,SignalRConfigurationValidator>());
```

Comment:
I'm going to leave this here (unrelated to this line) but i think we had missed another one (even in my PR) `UseSignalR` in https://github.com/OrleansContrib/SignalR.Orleans/blob/master/src/SignalR.Orleans/Extensions.cs#L33

Reply:
Oh! Yeah... the other storge provider... Yeah, I should validate that as well. Thanks!